docs/dm/zh_CN/maintenance/dm-migrate.md

```diff
@@ -0,0 +1,30 @@
+迁移 DM-worker
+===
+
+当 DM-worker 不可用时，需要将在不可用的 DM-worker 上运行的任务迁移到另一台 DM-worker上
+
+### 迁移流程
+1. 部署新的 DM-worker
+2. 确定不可用的 DM-worker 同步点
+2. 对新部署的 DM-worker 使用 `migrate-relay` 命令，接上一个 DM-worker 的同步点
+
+### 如何确定不可用的 DM-worker 同步点
+通常会保存在下游数据库里，task配置里的`meta-schema`字段指名了保存在下游数据库的位置。
+
+这里假设`meta_schema`的值是`dm_meta`， 任务名是`test`就到下游数据库中找到`dm_meta`这个库中，
+找到`test_syncer_checkpoint`这个表, 用SQL查询出所有的同步点
+`select binlog_name, binlog_pos from dm_meta.test_syncer_checkpoint where id = 要迁移的 DM-worker 对应的上游 MySQL 的 source-id and is_global = 1;` 
```

Comment:
do you want to show this `SELECT` statement in a separate line? maybe you need to add spaces at the tail of the line above or use ` ``` `to start a code block?

in addition, use some marker to wrap `要迁移的 DM-worker 对应的上游 MySQL 的 source-id` like `{要迁移的 DM-worker 对应的上游 MySQL 的 source-id}`?